app/config.py

```diff
@@ -120,7 +120,7 @@ class Config(object):
     NOTIFY_REQUEST_LOG_LEVEL = os.getenv("NOTIFY_REQUEST_LOG_LEVEL", "INFO")
 
     # Cronitor
-    CRONITOR_ENABLED = False
+    CRONITOR_ENABLED = os.environ.get("CRONITOR_ENABLED", False)
```

Comment:
Other settings are comparing with `0`/`1` to get their boolean result, this isn't.

Reply:
Good spot! I've updated.